assets/js/blocks/cart-checkout/checkout-i2/editor-utils.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { getBlockTypes } from '@wordpress/blocks';
+
+export const getRegisteredBlockNamesByParent = (
```

Comment:
Names are starting to getting out of hand. Just an observation.

Reply:
Resolved in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4684/commits/6a879e327188f968fac8c314d1d0f7ccfd566b86